### DISCUSSION TITLE
How to know which files have been edited of OpenAM to upgrade to AM?

### DISCUSSION THREAD

**Discussion by korah91** [OpenIdentityPlatform/OpenAM#815] [Q&A]
Hi,

Is there any recommended way to check for files of OpenAM that have been added/modified so that I can backup them and can import them into a new deployment of OpenAM or AM? 
My current deployment lacks documentation of such changes and I don't know how to start looking for these kind of files. As upgrading to AM is out of scope of this repo, I was wondering if you could help me knowing which files should I compare or check.

Thanks!

**Comment by vharseko** [comment]
[OpenAM: Access Management Upgrade Guide](https://doc.openidentityplatform.org/openam/upgrade-guide/)
